### PR TITLE
Adding Getting Started info to README

### PR DESCRIPTION
**Pull request by davidjoy**
Still working on the README, but figure I'll check things in as I go.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,28 +1,68 @@
...
+
+  ``git clone https://github.com/edx/frontend-app-payment.git``
+
+2. Install npm dependencies:
+
+  ``cd frontend-app-payment && npm install``
+
+3. Start the dev server:
+
+  ``npm start``
```
Comment (albemarle) [inline README.rst]: Should we link to devstack and mention that it's a prereq?

File: README.rst [hunk 2]
```diff
@@ -1,28 +1,68 @@
...
+must add it to the Purgecss configuration or it will be unstyled in the production build.
 
 
-Local Dev Server + Stage API
-----------------------------
+Appendix A: Local Dev Server + Stage API
+----------------------------------------
 
 If you would like to run this frontend against stage.edx.org you can run npm run start:stage and
 access your development server at https://local.stage.edx.org:1998 after the initial setup
```
Comment (albemarle) [inline README.rst]: Are there plans afoot to make stage accessible only from the VPN?

File: README.rst (hunk 1 above)
Comment (davidjoy) [reply README.rst]: ... right. :)

File: README.rst (hunk 2 above)
Comment (davidjoy) [reply README.rst]: Not sure.

**Pull request merged**